.env.template

```diff
@@ -27,6 +27,30 @@ MODELS=`[
       }
     ]
   },
+  {
+    "name" : "HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "tokenizer": "HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "description": "Zephyr 141B-A35B is a fine-tuned version of Mistral 8x22B, trained using ORPO, a novel alignment algorithm.",
+    "modelUrl": "https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "websiteUrl": "https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "logoUrl": "https://huggingface.co/datasets/huggingchat/models-logo/resolve/main/zephyr-logo.png",
+    "parameters": {
+      "truncate" : 24576,
+      "max_new_tokens" : 8192,
+    },
+    "promptExamples" : [
+      {
+        "title": "Write an email from bullet list",
+        "prompt": "As a restaurant owner, write a professional email to the supplier to get these products every week: \n\n- Wine (x10)\n- Eggs (x24)\n- Bread (x12)"
```

Comment:
```suggestion
        "prompt": "Write a poem to help me remember the first 10 elements on the periodic table, giving each element its own line."
```

Reply:
This is the output I get

In the realm of elements, a tale is told,                                                                                                                                                                 
Of atoms that make up our world, both new and old.                                                                                                                                                        
Their names and symbols, we shall now recite,                                                                                                                                                             
In a poetic form, to keep them in sight.                                                                                                                                                                  

Hydrogen, H, the lightest of them all,                                                                                                                                                                    
Found in water, it's the universe's call.                                                                                                                                                                 
Helium, He, the noble gas so fine,                                                                                                                                                                        
Floats balloons high, with its nature benign.                                                                                                                                                             

Lithium, Li, third in our elemental quest,                                                                                                                                                                
Light and reactive, in batteries it rests.                                                                                                                                                                
Beryllium, Be, a metal tough and strong,                                                                                                                                                                  
In alloys it shines, where it truly belongs.                                                                                                                                                              

Boron, B, a nonmetal with many a use,                                                                                                                                                                     
From glass to ceramics, its versatility loose.                                                                                                                                                            
Carbon, C, the backbone of life so dear,                                                                                                                                                                  
In diamonds and graphite, it's always near.                                                                                                                                                               

Nitrogen, N, the air we breathe and more,                                                                                                                                                                 
A crucial component, from plants to our core.                                                                                                                                                             
Oxygen, O, the gas that fuels our breath,                                                                                                                                                                 
Essential for life, till the hour of death.                                                                                                                                                               

Fluorine, F, a halogen, reactive and bold,                                                                                                                                                                
Found in toothpaste, its benefits untold.                                                                                                                                                                 
Neon, Ne, another noble gas to behold,                                                                                                                                                                    
In signs it glows, a sight to behold.                                                                                                                                                                     

These first ten elements, our world they shape,                                                                                                                                                           
From the smallest to the largest, their roles they drape.                                                                                                                                                 
So remember their symbols, their order and name,                                                                                                                                                          
In the periodic table, they'll always remain

Reply:
Oh I like this nice!